pydis_site/templates/events/pages/code-jams/8/_index.html

```diff
@@ -20,10 +20,54 @@
         and walking through the program that your team has created.
     </p>
 
+    <h3 id="winners"><a href="#winners">Code Jam Winners</a></h3>
+        <p>Congratulations to our winners and the two runner ups! Check out their projects below.</p>
+        <h4><i class="fa fa-trophy"></i> Perceptive Porcupines: WTPython!?</h4>
+        <p><em>VV, Poppinawhile, ethansocal, Jeff Z, Cohan, ¯\_(ツ)_/¯</em></p>
+        <p>
+            What the Python (wtpython) is a simple terminal user interface that allows you to explore relevant answers on Stackoverflow without leaving your terminal or IDE. When you get an error, all you have to do is swap python for wtpython. When your code hits an error, you'll see a textual interface for exploring relevant answers allowing you to stay focused and ship faster!
+            <br/>
+            <a href="https://www.youtube.com/watch?v=DV3uMdsw9KE" title="Perceptive Porcupines Demo Video" target="_blank" rel="noopener"><i class="fa fa-video"> </i> Demo video</a>
+            <br/>
+            <a href="https://github.com/what-the-python/wtpython" title="Perceptive Porcupines GitHub Repository" target="_blank" rel="noopener"><i class="fa fa-github"></i> Githib Repository</a>
+            <br/>
+
+        </p>
+
+        <h4><i class="fa fa-medal"></i> Lovable Lobsters: Ultimate Tic Tac Toe</h4>
+        <p><em>A5Rocks, Bast, Dacheat, mega_hirtz, CopOnTheRun, richphi</em></p>
+        <p>
+            Thinking inside a box, that is inside a box, that is inside yet another box.
+
+            The terminal program created by the Lovable Lobsters allows you to play Ultimate Tic Tac Toe right form your terminal. The really impressive part though? You can play with your friends and family over your network! Their program has a server-client set-up that lets you play with your friends and family from different computers.
+            <br/>
+            <a href="https://www.youtube.com/watch?v=WI9tgQeAfXw" title="Lovable Lobsters Demo Video" target="_blank" rel="noopener"><i class="fa fa-video"> </i> Demo video</a>
+            <br/>
+            <a href="https://github.com/A5rocks/code-jam-8" title="Lovable Lobsters GitHub Repository" target="_blank" rel="noopener"><i class="fa fa-github"></i> Githib Repository</a>
```

Comment:
```suggestion
            <a href="https://github.com/A5rocks/code-jam-8" title="Lovable Lobsters GitHub Repository" target="_blank" rel="noopener"><i class="fa fa-github"></i> Github Repository</a>
```
Multiple instances of this throughout the page as far as I can tell~

Reply:
;-; ... You know, you'd think I'd be able to spell GitHub correctly at this point.